src/components/ENSAddress.tsx

```diff
@@ -28,18 +28,35 @@ function ENSAddressSuspended({
   )
 }
 
-export default memo<ENSAddressProps>(({ address, network }) => {
-  const { md, lg } = useBreakpoints()
-  const truncateSize = md ? (lg ? 25 : 17) : 11
-
+function ENSAddress({
+  address,
+  network,
+  truncateSize,
+}: ENSAddressProps & { truncateSize: number }) {
   return (
     <Suspense fallback={truncateMiddleIfNeeded(address, truncateSize)}>
       <ENSAddressSuspended
+        truncate
         address={address}
-        truncateSize={truncateSize}
-        truncate={!lg}
         network={network}
+        truncateSize={truncateSize}
       />
     </Suspense>
   )
+}
+
+export default memo<ENSAddressProps>(({ address, network }) => {
+  return (
+    <span>
+      <span className={display('block', 'md:hidden')}>
```

Comment:
Maybe we should extract those styles as well? Something like
`displayOnMobiles = (box: 'block', 'flex', 'inline') => dispaly(box, 'md:hidden)`
?

Reply:
`box` should inherit type from tailwind

Reply:
Oh, you do have `displayToMd`, maybe you can use it here?

Reply:
Changed to `displayToMd`